src/components/images/images.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {gql} from 'apollo-boost';
+import {useQuery} from 'react-apollo';
+
+const GET_IMAGES = gql`
+query {
+    allImages {
+      id
+      image_url
+    }
+  }
+`
+
+export const Images = () => {
+    const{loading, error, data} = useQuery(GET_IMAGES)
+   
+    if (loading) {
+        return (
+            <div>Loading...</div>
+        )
+    } 
+    if (error) {
+        return (
+            <div>Error...</div>
+        )
+    } 
+    if (data) {
+        // using this instead of useState as the latter causes an infinite loop
+        const images = data.allImages
+        
+        return (
+        <>
+        {images.map((item) => (
```

Comment:
Just a note for later... when we pull in images for the actual art views we'll want to make sure to set an alt tag... maybe the name of the piece?

Reply:
@Memitaru Agreed. This is just for testing purposes to confirm we can talk to the database. In future work we will need this and other semantic HTML for accessibility needs.